### ISSUE TITLE
Fail to create models with EfficientNetB*-backbone.

### ISSUE THREAD

**Issue by simon-larsson** [qubvel/segmentation_models#198]
I tried Unet with different some different backbones and had no problems until i tried variations of efficientnet.

I tried with both 0.2.* versions and 1.0.* versions.

**Code:**
```
import segmentation_models as sm

model = sm.Unet('efficientnetb0', 
                classes=4, 
                activation='sigmoid', 
                input_shape=(256, 256, 3), 
                encoder_weights='imagenet')
```

**Error:**
```
~\AppData\Local\Continuum\anaconda3\lib\site-packages\segmentation_models\utils.py in wrapper(*args, **kwargs)
     29                     kwargs[new_arg] = kwargs[old_arg]
     30 
---> 31             return func(*args, **kwargs)
     32 
     33         return wrapper

~\AppData\Local\Continuum\anaconda3\lib\site-packages\segmentation_models\unet\model.py in Unet(backbone_name, input_shape, classes, activation, encoder_weights, encoder_freeze, encoder_features, decoder_block_type, decoder_filters, decoder_use_batchnorm, **kwargs)
     61                             input_tensor=None,
     62                             weights=encoder_weights,
---> 63                             include_top=False)
     64 
     65     if encoder_features == 'default':

~\AppData\Local\Continuum\anaconda3\lib\site-packages\segmentation_models\backbones\__init__.py in get_backbone(name, *args, **kwargs)
     73 
     74 def get_backbone(name, *args, **kwargs):
---> 75     return Classifiers.get_classifier(name)(*args, **kwargs)
     76 
     77 

~\AppData\Local\Continuum\anaconda3\lib\site-packages\classification_models\__init__.py in get_classifier(cls, name)
     84     @classmethod
     85     def get_classifier(cls, name):
---> 86         return cls._models.get(name)[0]
     87 
     88     @classmethod

TypeError: 'NoneType' object is not subscriptable
```

**Comment by qubvel** [comment]
Hi, please check version consistency with this code (libs should be both 1* or 0* versions):
```
import segmentation_models as sm
import efficientnet
print(sm.__version__)
print(efficientnet.__version__)

model = sm.Unet('efficientnetb0', 
                classes=4, 
                activation='sigmoid', 
                input_shape=(256, 256, 3), 
                encoder_weights='imagenet')
```

**Comment by simon-larsson** [comment]
I got it working with the 1* dependencies now. I think I previously got the error from not reloading my notebook kernel when switching from 0*.

I still cannot manage to get it working with 0*. 

My versions:

Name: segmentation-models
Version: 0.2.1

Name: image-classifiers
Version: 0.2.0

Name: efficientnet
Version: 0.0.4

However, I am happy to continue with 1* since it is working. So you can close this if this does not help you in anyway. :)

**Comment by maremoto** [comment]
Hi, I have exactly the same issue, in Windows, 
the versions of the packages are:

conda 4.3.21 (I use a  conda environment)
python 3.6.7
jupyter 4.4.0
tensorflow-gpu 1.14.0
keras 2.3.0
segmentation_models 0.2.1
image-classifiers 0.2.0
efficientnet 0.0.4
albumentations 0.3.3

(if I try to execute efficientnet.__version__ in the notebook I get 
`AttributeError: module 'efficientnet' has no attribute '__version__'`
)

There is no way of creating an efficientnet model, it always fails with 

```
lib\site-packages\classification_models\__init__.py in get_classifier(cls, name)
     84     @classmethod
     85     def get_classifier(cls, name):
---> 86         return cls._models.get(name)[0]
     87 
     88     @classmethod

TypeError: 'NoneType' object is not subscriptable
```

Thanks.

**Comment by simon-larsson** [comment]
Upgrading to pre-released versions worked for me. You can install with:

```
$ pip install -U --pre segmentation-models
$ pip install -U --pre efficientnet
```

**Comment by maremoto** [comment]
Hi,

Thanks a lot for your response, I managed to get it working with pre-released versions, and I ran into #209 issue, but it is not a problem because I've manually modified objects.py for a valid __name__ creation, while the patch goes into release.

Thanks again,
Mar

**Comment by gkaissis** [comment]
It's not working with the pre-release versions for me. It creates trouble with segmentation_models (ImportError)...
